README.rst

```diff
@@ -47,9 +55,41 @@ SciPy a try!
 
 For the installation instructions, see INSTALL.rst.txt_.
 
-We appreciate and welcome contributions. If you would like to take part in
-SciPy development, take a look at the file CONTRIBUTING.rst_.
+
+Call for Contributions
+----------------------
+
+The SciPy project welcomes your expertise and enthusiasm!
+
+Small improvements or fixes are always appreciated; issues labeled as "good
+first issue" may be a good starting point. If you are considering larger
+contributions to the source code, please contact us through the [mailing
+list](https://scipy.org/scipylib/mailing-lists.html) first.
```

Comment:
These are Markdown-style links in a `.rst` file, they don't work (a few more further down).

Reply:
🤦  indeed! I will fix these.